web/app.js

```diff
@@ -811,9 +811,7 @@ const PDFViewerApplication = {
    */
   _hideViewBookmark() {
     // URL does not reflect proper document location - hiding some buttons.
-    const { toolbar, secondaryToolbar } = this.appConfig;
-    toolbar.viewBookmark.hidden = true;
-    secondaryToolbar.viewBookmarkButton.hidden = true;
+    this.appConfig.viewBookmarkButton.hidden = true;
```

Comment:
I believe that this will throw, since it's not *exactly* what I suggested in https://github.com/mozilla/pdf.js/pull/15413#discussion_r965615549

Reply:
pfff... my bad.